tests/js/spec/views/settings/organizationIntegrations/__snapshots__/index.spec.jsx.snap

```diff
@@ -127,6 +132,11 @@ exports[`OrganizationIntegrations render() with installed integrations Displays
             },
             "author": "Rick",
             "description": "*markdown* formatted Jira _description_",
+            "features": Array [
+              Object {
+                "description": "*markdown* feature description",
+              },
+            ],
```

Comment:
Should any of the tests use disabled gates to verify the disabled button paths?

Reply:
eedc733